apps/api/src/app/rate-limiting/usecases/get-api-rate-limit-maximum/get-api-rate-limit-maximum.usecase.ts

```diff
@@ -48,7 +49,9 @@ export class GetApiRateLimitMaximum {
     }
 
     let apiRateLimits: IApiRateLimitMaximum;
+    let apiServiceLevel: ApiServiceLevel;
     if (environment.apiRateLimits) {
+      apiServiceLevel = CUSTOM_API_SERVICE_LEVEL;
```

Comment:
When we have a custom rate limit set on the environment, return a constant with value of `'custom'` to enable metrics reporting on use of custom rate limits, and return this info to API client.